be/src/storage/olap_common.h

```diff
@@ -376,7 +376,17 @@ struct Version {
 
     bool contains(const Version& other) const { return first <= other.first && second >= other.second; }
 
-    bool operator<(const Version& rhs) const { return second < rhs.second; }
+    bool operator<(const Version& rhs) const {
+        if (second < rhs.second) {
```

Comment:
```suggestion
        if (second == rhs.second) {
               return first < rhs.first;
        } else {
               return second < rhs.second;
        }
```

Reply:
Maybe it is not that clear about what we want to do?